common/app/templates/headerInlineJS/bootSystemJS.scala.js

```diff
@@ -95,8 +95,8 @@ System['import']('core').then(function () {
                     images.listen();
 
                     if (guardian.isModernBrowser) {
-                        alreadyVisted = storage.local.get('alreadyVisited') || 0;
-                        storage.local.set('alreadyVisited', alreadyVisted + 1);
+                        alreadyVisted = storage.local.get('gu.alreadyVisited') || 0;
```

Comment:
Should we use the name `gu.already.visited`, in the convention set by the likes of `gu.prefs.messages`, `gu.history.popular`, et cetera?